interface/resources/qml/hifi/dialogs/security/EntityScriptQMLWhitelist.qml

```diff
@@ -129,30 +167,39 @@ Rectangle {
           id: notificationText;
           text: ""
           // Text size
-          size: 14;
+          size: 16;
           // Style
           color: "white";
           elide: Text.ElideLeft;
           // Anchors
-          anchors.right: parent.right;
-          anchors.rightMargin: 130;
+          anchors.right: parent.left;
+          anchors.rightMargin: 10;
       }
     }
     
     HifiStylesUit.RalewayRegular {
         id: descriptionText;
-        text: "Separate your URLs by line, not commas. Example:
-        https://google.com/
-        https://bing.com/
-        https://mydomain.here/
-        \nEnsure there are no spaces or whitespace.
-        \nFor QML files, you can only whitelist each file individually 
-        ending with '.qml'."
+        text: 
+"The whitelist checks scripts/QML as it is loaded.<br/>
+Therefore, if a script is cached or has no reason to load again,<br/>
+then removing it from the whitelist will not be effective until<br/>
+it is reloaded.<br/>
+Separate your whitelisted domains by line, not commas. e.g.
+<blockquote>
+	<b>https://google.com/</b><br/>
+	<b>hifi://the-spot/</b><br/>
+	<b>127.0.0.1</b><br/>
+	<b>https://mydomain.here/</b>
+</blockquote>
+Ensure there are no spaces or whitespace.<br/><br/>
+For QML files, you can only whitelist each file individually <br/>
```

Comment:
For QML files, you can only whitelist each file individually,<br/>

Reply:
Not sure if there's a diff?